cocotb/share/makefiles/simulators/Makefile.modelsim

```diff
@@ -31,5 +31,4 @@
 #ARCH:=i686
 
 # Everything else is identical to Quest
-include $(COCOTB_SHARE_DIR)/makefiles/simulators/Makefile.questa
-
+include $(COCOTB_MAKEFILES_DIR)/simulators/Makefile.questa
```

Comment:
Where is this variable defined?

Reply:
In `Makefile.sim` can also use `cocotb-config`

Reply:
I think invoking `cocotb-config`, or using a relative path, would be clearer.